NumpyXBench/operators/common_ops.py

```diff
@@ -43,14 +52,11 @@ def get_forward_func(self):
         """
         Get the forward function of the Op.
         """
-        if self._backend in ['numpy', 'np']:
-            module = sys.modules['numpy']
-        elif self._backend in ['mxnet', 'mx']:
-            module = sys.modules['mxnet.numpy']
-        elif self._backend in ['pytorch', 'torch']:
-            module = sys.modules['torch']
-        else:
-            raise NotImplementedError("Backend not supported now!")
+        try:
+            module = sys.modules[module_switcher[self._backend]]
+        except ValueError:
+            raise NotImplementedError(f'Backend: {self._backend} not supported now!')
```

Comment:
ok, change to 'not support or backend not installed'

Reply:
Done